dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -119,4 +119,9 @@
         <modifyDataType columnName="forumurl" newDataType="varchar(256)" tableName="service"/>
         <modifyDataType columnName="forumurl" newDataType="varchar(256)" tableName="workflow"/>
     </changeSet>
+    <changeSet author="nzhao (generated)" id="toolForumUrl">
```

Comment:
I think this should be part of a new migrations.1.11.0.xml, unless we're targeting for a hotfix. Adding a new migrations file is a bit of a pain; you also have to at least update compose_setup, propose_migrations.sh, and the integration tests. Just pointing it out, will defer to @denis-yuen  and @garyluu

Reply:
ya, I agree with 1.11.0